src/checker.cpp

```diff
@@ -2028,6 +2028,8 @@ gb_internal void add_type_info_type_internal(CheckerContext *c, Type *t) {
 		break;
 
 	case Type_Struct:
+		if (bt->Struct.fields_wait_signal.futex.load() == 0)
+			return;
```

Comment:
This "might" work, but I worried it'll break something later. But maybe that is the better solution at any rate.